packages/help-center/src/components/help-center-notice.tsx

```diff
@@ -63,12 +64,43 @@ export function HelpCenterOwnershipNotice( {
 		}
 		return null;
 	}
+
+	return <HelpCenterNotice>{ responses[ ownershipResult ] }</HelpCenterNotice>;
+}
+
+export function HelpCenterActiveTicketNotice( {
+	tickets,
+}: {
+	tickets: SupportTicket[] | undefined;
+} ) {
+	if ( ! tickets || ! tickets.length ) {
+		return null;
+	}
+
+	return (
+		<HelpCenterNotice>
+			<p>
+				<strong>
+					{ sprintf(
+						/* translators: %s humanized date ex: 2 hours ago */
+						__( 'You submitted a request %s.' ),
```

Comment:
We should use the domain, like `__( 'You submitted a request %s.', __i18n_text_domain__)`